lib/game.rb

```diff
@@ -1 +1,22 @@
-puts "Welcome to Tic-Tac-Toe!"
\ No newline at end of file
+class Game
+    def initialize
+        puts "Welcome to Tic-Tac-Toe!"
+    end
+
+    def print_board
+       puts "
+             |       | 
+         1   |   2   |   3
+       _____ | _____ | _____
+             |       |  
+         4   |   5   |   6
+       _____ | _____ | _____
+             |       | 
+         7   |   8   |   9
+             |       | 
+        "
+    end
+end
+
+g = Game.new()
+g.print_board
```

Comment:
What are the responsibilities of the Game class?  Do you think it has more than one responsibility?  How can we follow SOLID principles and encapsulate responsibilities?